pontoon/sync/vcs/models.py

```diff
@@ -206,17 +207,19 @@ def locale_directory_paths(self):
         Create locale directory, if not in repository yet.
         """
         locale_directory_paths = {}
+        parent_directories = set()
```

Comment:
Sometimes locales can actually be on different paths - if we use multiple repositories (e.g. in Firefox we have locales on `hg.mozilla.org` and `BitBucket`). So we store `parent_direcotry` for each of them.